submodules/wizard/wizard.js

```diff
@@ -159,7 +156,15 @@ define(function(require) {
 						accessLevel: 'full',
 						allowedAppIds: []
 					}
-				},
+				};
+
+			if (!_.chain(monster.config).get('whitelabel.realm_suffix').isEmpty().value()) {
+				defaultData.generalSettings.accountInfo.whitelabeledAccountRealm = monster.util.randomString(7) + '.' + monster.config.whitelabel.realm_suffix;
```

Comment:
I assume this follows the same pattern than in the old account creation wizard?

Reply:
Yes, it follows the same pattern: https://github.com/2600hz/monster-ui-accounts/blob/master/app.js#L178-L180